### PR TITLE
docs: talked about swagger in a slightly funny way

### PR DESCRIPTION
**Pull request by philsturgeon**

### CODE REVIEW

File: netlify/subpages/api-design-guide.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ tabs:
     title: API Design Basics
   - href: /api-design-guide/tooling
     title: API Design Tooling
-  - href: /api-design-guide/oas-spec
+  - href: /api-design-guide/OpenAPI-spec
```
Comment (tbarn) [inline netlify/subpages/api-design-guide.md]: We have to be careful about changing URLs for SEO reasons. What do you think @moaiandin?

Comment (marbemac) [reply netlify/subpages/api-design-guide.md]: can we 301 permanent redirect old to new? Also suggest keeping routes lowered, e.g. `/openapi-spec`.

File: netlify/subpages/api-design-guide.md [hunk 2]
```diff
@@ -40,24 +40,24 @@ There is not a single approach to design APIs “the right way.” Instead, we n
...
+OpenAPI allows you to define how your REST API works, in a way that can be easily consumed by both humans and machines. It serves as a contract that specifies how a consumer can use the API and what responses you can expect.
 
-OAS 3.0 was released in July, 2017, by the OpenAPI Initiative, a consortium of member companies who want to standardize how REST APIs are described. There are various other approaches to API description:
+OpenAPI v3.0 was released in July, 2017, by the OpenAPI Initiative, a consortium of member companies who want to standardize how REST APIs are described. There are various other approaches to API description:
 
-- **OAS 2.0**, based on the Swagger definition. Still widely used, but being replaced by OAS 3.0
-- **Swagger**, the best known of the approaches being replaced or augmented by OAS 3.0
+- **OpenAPI v2**, (formerly known as Swagger v2.0) is still widely used, but increasingly being replaced by OpenAPI v3
+- **JSON Schema**, very similar to OpenAPI, but able to describe any JSON-like data, not just APIs
```
Comment (marbemac) [inline netlify/subpages/api-design-guide.md]: I think it's important to distinguish the primary use case for x (openapi versus y (json schema). Otherwise folks are like "what do I use?!" 🤷‍♂. Open to ideas, but I think of JSON Schema as being great for Modeling, while OpenAPI is great for API Design.

Comment (tbarn) [reply netlify/subpages/api-design-guide.md]: People use design and modeling synonymous

Comment (philsturgeon) [reply netlify/subpages/api-design-guide.md]: Yeah they're used interchangeably. I don't know what the difference is between design and modeling.

API Design is the act of deciding what your service model and data model will look like.

Comment (philsturgeon) [reply netlify/subpages/api-design-guide.md]: We use them interchangeably here at stoplight too btw.

File: netlify/subpages/api-design-guide.md (hunk 1 above)
Comment (philsturgeon) [reply netlify/subpages/api-design-guide.md]: that was an accident but yeah OAS is not something I ever see used outside of stoplight, and OpenAPI is picking up steam as a keyword. We should use OpenAPI or openapi whenever possible.

Comment (philsturgeon) [reply netlify/subpages/api-design-guide.md]: I've pushed updates making this consistent in all the places

Comment (philsturgeon) [reply netlify/subpages/api-design-guide.md]: (not sure how to do redirects though)

Comment (lottamus) [reply netlify/subpages/api-design-guide.md]: You can add 301 redirects here: https://github.com/stoplightio/stoplight.io/blob/develop/public/_redirects

Comment (lottamus) [reply netlify/subpages/api-design-guide.md]: Here are the netlify docs: https://www.netlify.com/docs/redirects/

Comment (philsturgeon) [reply netlify/subpages/api-design-guide.md]: Cool yep all good with Netlify, I forgot we were on that. :D

File: netlify/subpages/api-design-guide.md (hunk 2 above)
Comment (philsturgeon) [reply netlify/subpages/api-design-guide.md]: Also with JSON Schema offering JSON HyperSchema you could use OpenAPI or JSON Schema interchangeably, two different ways to solve the same problem. OpenAPI tackling a more RPCish "paths are super important" approach to HTTP APIs, JSON HyperSchema going for Hypermedia APIs. 

I definitely don't want to get into any of that here, but could somewhere, sometime.

File: public/_redirects [hunk 3]
```diff
@@ -9,16 +9,17 @@ https://stoplightio.netlify.com/* https://stoplight.io/:splat 301!
 /twitter https://twitter.com/stoplightio
 
 # Internal redirects
-/platform/prism /mocking/
```
Comment (tbarn) [inline public/_redirects]: Any reason why you remove the trailing `/`?

File: netlify/subpages/api-design-guide-oas-spec.md [hunk 4]
```diff
@@ -1,64 +0,0 @@
----
```
Comment (tbarn) [inline netlify/subpages/api-design-guide-oas-spec.md]: Why did you delete this page? Did you mean to replace it?

**Comment by tbarn** [comment]
Let's try to wrap this one up Tuesday since I will be linking to it from one of the landing pages and need this merged first.

File: netlify/subpages/api-design-guide-oas-spec.md (hunk 4 above)
Comment (philsturgeon) [reply netlify/subpages/api-design-guide-oas-spec.md]: Absolutely, it was sitting around locally untracked 🤦‍♂

File: public/_redirects (hunk 3 above)
Comment (philsturgeon) [reply public/_redirects]: Yeah it's inconsistent all over the place, we should pick one and go with it. Without is the most common, and with doesnt actually add anything.

**Comment by philsturgeon** [comment]
@tbarn fixed and ready to go!

File: public/_redirects (hunk 3 above)
Comment (tbarn) [reply public/_redirects]: I checked on what we've done in the past and we want forward slashes on redirected pages. It is better for SEO.

**Comment by tbarn** [comment]
@philsturgeon left a comment on the slash question and then we should be ready to go.

**Comment by philsturgeon** [comment]
@tbarn fixed!

**Comment by philsturgeon** [comment]
@lottamus lets get this in before we start press

**Comment by philsturgeon** [comment]
@marbemac @rossmcdonald bump :D
